### ISSUE TITLE
THE ABSOLUTE STATE OF MOTHS, or, how I learned to stop shitposting and finally make a formal complaint about the Poojmoth

### ISSUE THREAD

**Issue by tiramisuapimancer** [citadel-station/citadel-main#13222]
**Formally, this is a feature request. However, it contains something of a bug report, in that some content complained about does not function as what I could fathom as in the intended manner. To the maintainers of this codebase, I dearly request that you consider this to be a _bug request_. I am requesting moths. Bring them back.**

# The Problem With Anthropomorphic Insect 
Simply put, moths on the Citadel codebase are in a disgusting state, left in it to rot after the otherwise much-needed species squash PR. The species squash PR existed in order to merge together the redundant Mammal clones together into Anthropomorph.  For whatever reason, this same PR decided to remove the Moth species and replace it with... Replace it with _a redundant clone of Mammal._ 

Here's what a Moth looked like before they were removed from Citadel, in a screenshot hastily taken from Beestation.
![image](https://user-images.githubusercontent.com/32314478/90773051-d5097400-e2a1-11ea-8771-d0c1060e39cc.png)

And now here's what the same Moth looks like, using the Insect species on Citadel.
![image](https://user-images.githubusercontent.com/32314478/90773209-0c782080-e2a2-11ea-84c8-33921f0a5610.png)

The difference is striking, but one could simply say "deal with it", say they like the current head better, etc. That's fine, Citadel is all about options, right? Maybe a lot of people actually do like the alternate head better than the little plus eyes. Except... both before the change to insect, and now, Moths, Insects, and all the Mammal clones that were merged into Anthropomorph, have been able to select snout options, which includes the Bug snout. The Bug snout option is nearly identical to the head on the Insect species, except you can color the eyes using Tertiary color. Here's a Bug snout on aforeposted Example Moth Which Totally Isn't My Main Static, with eye color adjusted to be more noticeable.
![image](https://user-images.githubusercontent.com/32314478/90774237-97a5e600-e2a3-11ea-8a9b-4b109123c778.png)

So the snout is significantly better in most ways besides in that weird little black pixel that's in the corner. What is that, anyways? Oh, yeah, it's the eye pixel. You *can* adjust the eye color on an Insect without using the Bug snout that is otherwise identical to their head sprite, actually. I technically fibbed. Here's an Insect without a snout selected and red eyes.
![image](https://user-images.githubusercontent.com/32314478/90774541-1ef35980-e2a4-11ea-9846-99e006ba9078.png)

What the fuck is that? What the fuck. ~~(Yeah, moths actually had this issue before the Insect change. You could change the eye color and it would just change a single pixel in the middle of the +-shaped eye. This is where the eye pixel is on single-pixel-eyed sorts. Anyways!)~~

There are a couple other issues with the Insect species and the Moth parts cannibalized in its name. Here's an Insect with blue fluff.
![image](https://user-images.githubusercontent.com/32314478/90775023-c7a1b900-e2a4-11ea-8065-c462f102ef51.png)
There are no antennae sprites for the back view. Also, the bug snout has antennae of its own that don't quite match up with the antennae attached to the fluff. Also, the missing back antenna isn't specific to the Insect and can be witnessed with any species, hell, even better, now that you can select it on species without antennae in their sprite. Here's an Anthropomorph with Reddish fluff, with the lack of antenna sprites on the back circled. 
![image](https://user-images.githubusercontent.com/32314478/90776466-95915680-e2a6-11ea-95f0-440a6a69f35e.png)

Also, there seems to be a weird thing underneath the fluff that's inherent to the Insect sprite. To showcase, here's a screenshot of an Insect colored white, without fluff, snout, or antennae. 
![image](https://user-images.githubusercontent.com/32314478/90775826-cb820b00-e2a5-11ea-91cb-0eed7e047637.png)
I'm genuinely not sure what this is supposed to be. It looks weird as all fuck with the moth fluff over it and it looks weird as fuck without anything to cover it up. It's also colored brown which makes it look like shit if the moth is trying to be some other color as exampled here. 
![image](https://user-images.githubusercontent.com/32314478/90776135-32072900-e2a6-11ea-9826-4e7b0d2f2ead.png)

Anyways, back to the chase- here's a moth character created using the Anthropomorph species.
![image](https://user-images.githubusercontent.com/32314478/90776991-51528600-e2a7-11ea-86af-56bd34865af8.png)
You can get everything using Anthro's options- Anthro has sufficient support for creating an insectoid character, down to the wings, fluff, the meat, the voice, snout, markings, even bug 'tails' that you can't really see underneath the moth wings. In my opinion this genuinely looks marginally better than just using an Insect.

Things Anthro doesn't have that Insect does... The markings from the moths that don't precisely match up with the body shape of either, and a few gameplay differences that were previously unique to Moths (though a few gameplay features of Moths were removed during the Insectification, such as flashability).

Anyways, I'm losing the steam I had when I started this, so it's time to wrap it up.

# TL;DR
Moths were replaced with a worse clone of Mammal, Insect, in the same PR that existed to remove useless clones of Mammal. There is next to no reason to use the Insect species over the Anthro species, and with its existence removed the actually nice looking, gameplay unique Moth species. The Insect species is full of visual gore that I pray is unintentional, including the moth fluff sprites, which have no back antennae for some reason, even on other species. 

~~Come on, Skyrat even has both Moth and and Insect, for the insane bastards who would actually use the mess that is the Insect species. But I'm not going on fucking Skyrat.~~

**Comment by Ludox235** [comment]
poojmoths were a mistake

**Comment by timothyteakettle** [comment]
ill have a look at what the main differences are between them in terms of code at some point
right now i'm focusing on some other customization stuff but i'll add this to the list

the likely solution to this is to make it so your body sprite id is saved as a preference and not a hardset thing per species, and let anthromorphs choose between all body sprites including the old moth ones

**Comment by tiramisuapimancer** [comment]
How about the gameplay differences?

**Comment by timothyteakettle** [comment]
to my knowledge the only gameplay difference was being able to move in space with their wings when there's also pressure
not a good idea personally however i can fix the sprites up as i've now coded the ability to swap body sprites
